src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseError.cs

```diff
@@ -3,38 +3,139 @@
 
 namespace System.Text.RegularExpressions
 {
-    internal enum RegexParseError
+    /// <summary>
+    /// Contains detailed information on a regular expression parse error, used in <see cref="RegexParseException.Error"/>.
+    /// This information can be used to give a more meaningful error to the user.
```

Comment:
The second sentence here would be better in remarks or deleted.

Reply:
@stephentoub, if you could do the honors, please do, I'm on mobile atm.